cnd/src/http_api/routes/rfc003/handlers/post_swap.rs

```diff
@@ -21,6 +24,72 @@ use futures_core::future::TryFutureExt;
 use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 
+async fn initiate_request<AL, BL, AA, BA>(
```

Comment:
Was that re-ordering necessary?

Reply:
Continuing on the discussion above, I'd be fine with the re-ordering if it was in a separate patch.  It adds unnecessary cognitive load to the reviewer having refactor done within a patch that does logic changes also.

Reply:
This re-ordering was absolutely not done in purpose. I have no idea how this happened.